OSDOCS-17079-3:CQA 2.0 for SUPP-1: Remote Health Monitoring (Insight (#114934)
openshift/openshift-docs: modules/additional-details-about-how-remote-health-monitoring-data-is-used.adoc

For more information about date collected to enable remote health monitoring, see "Information collected by Telemetry" and "Information collected by the {insights-operator}".

Red Hat collects data about your use of the Red Hat Product(s) for purposes such as providing support and upgrades, optimizing performance or configuration, minimizing service impacts, identifying and remediating threats, troubleshooting, improving the offerings and user experience, responding to issues, and for billing purposes if applicable.

Collection safeguards::
+
Red Hat employs technical and organizational measures designed to protect the telemetry and configuration data.

Sharing::
+
Red{nbsp}Hat might share the data collected through Telemetry and the {insights-operator} internally within Red Hat to improve your user experience. Red{nbsp}Hat might share telemetry and configuration data with its business partners in an aggregated form that does not identify customers to help the partners better understand their markets and their customers' use of Red{nbsp}Hat offerings or to ensure the successful integration of products jointly supported by those partners.

Third parties::
+

> Red Hat may engage certain third parties to assist in the collection, analysis, and storage of the Telemetry and configuration data.

Reviewer: All "Red Hat" instances need {nbsp} between them. There are a few above this one I didn't call out.

```suggestion
Re{nbsp} Hat might engage certain third parties to assist in the collection, analysis, and storage of the Telemetry and configuration data.
```

ifndef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
User control / enabling and disabling telemetry and configuration data collection::
+

> You may disable {product-title} Telemetry and the {insights-operator} by following the instructions "Remote health reporting".

Reviewer: ```suggestion
You can disable {product-title} Telemetry and the {insights-operator} by following the instructions "Remote health reporting".
```

Author: :heavy_check_mark:
